DEVELOPMENT.md

```diff
@@ -38,6 +38,18 @@ The example is from `2021\euc_counties.csv` which shows the structure :-
 |KY        |clay       |
 |KY        |clay       |
 
+#### Common Gotchas:
+- The fact sheet is often riddled leading and trailing extra spaces, tabs, new
+lines, etc.  These need to be stripped in the final `euc_counties.csv` file.
+- Louisiana calls their county equivalents a `parish`.  The fact sheet may
+report just the parish name, but the Census data marks the areas as parishes.
+So you must manually add `parish` to any Louisiana 'counties', eg
+`allen -> allen parish`.
```

Comment:
I looked into this Alaska actually calls some of their counties `boroughs`.  However in the crosswalk files the Alaskan county names do not include the word `borough` 🤷 .  I can't say for certain what will happen if there is a FEMA EUC (like from a 🌋  or something) in AK since we have never had one, but my rough first check suggests their won't be a problems.

Reply:
coolio yeah that was just what google told me when I looked up parishes lol. All good